lib/factor/factorQuadratic.js

```diff
@@ -225,19 +227,44 @@ function factorSumProductRule(node, symbol, aValue, bValue, cValue, negate) {
     for (const pair of factorPairs) {
       if (pair[0] + pair[1] === bValue) {
         // To factor, we go through some transformations
-        // TODO: these should be actual substeps
-        // 1. Break apart the middle term into two terms using our factor pair (p and q):
-        //    e.g. ax^2 + bx + c -> ax^2 + px + qx + c
+        // TODO: these should be actual substeps *alaina*
+        // 1. Break apart the middle term into two terms using our factor pair
+        //    (p and q): e.g. ax^2 + bx + c -> ax^2 + px + qx + c
         // 2. Consider the first two terms together and the second two terms
         //    together (this doesn't require any actual change to the expression)
         //    e.g. first group: [ax^2 + px] and second group: [qx + c]
         // 3. Factor both groups separately
         //    e.g first group: [ux(rx + s)] and second group [v(rx + s)]
         // 4. Finish factoring by combining the factored terms through grouping:
         //    e.g. (ux + v)(rx + s)
+        const substeps = [];
+        let status;
+
+        // 1. Break apart the middle term into two terms using our factor pair
+        // (p and q)
         const pValue = pair[0];
         const qValue = pair[1];
 
+        // SUBSTEP: ax^2 + bx + c -> ax^2 + px + qx + c
+        const sq = Node.Creator.constant(2);
+        const a = Node.Creator.constant(aValue);
+        const b = Node.Creator.constant(bValue);
+        const c = Node.Creator.constant(cValue);
+        const p = Node.Creator.constant(pValue);
+        const q = Node.Creator.constant(qValue);
+        const ax2 = Node.Creator.polynomialTerm(symbol, sq, a);
+        const px = Node.Creator.polynomialTerm(symbol, null, p);
+        const qx = Node.Creator.polynomialTerm(symbol, null, q);
+        newNode = Node.Creator.operator('+', [ax2, px, qx, c], true);
+        status = Node.Status.nodeChanged(
+          ChangeTypes.SIMPLIFY_ARITHMETIC, node, newNode);
```

Comment:
I'd make a new change type for this since it's a pretty unique use case and it'd be nice to describe it more accurately :)